src/webui/api/appcontroller.cpp

```diff
@@ -141,6 +141,8 @@ void AppController::preferencesAction()
     data[u"scan_dirs"_qs] = nativeDirs;
     // === END DEPRECATED CODE === //
 
+    // Excluded file names
+    data[u"excluded_file_names_enabled"_qs] = session->isExcludedFileNamesEnabled();
```

Comment:
Please bump the web api version too.

Reply:
It was bumped in [#17106](https://github.com/qbittorrent/qBittorrent/pull/17106/files#diff-13474ccd513f95826cbe8c120f2b739c0461d6d05a67fbaf6be8143a6f9d55c7R51)

Reply:
This PR introduces new setting fields so it should be bump again.

Reply:
All comments addressed.